example/spec/helpers/granuleUtils.js

```diff
@@ -145,29 +144,20 @@ async function setupTestGranuleForIngest(bucket, inputPayloadJson, granuleRegex,
 }
 
 const deleteGranules = async (prefix, granules) => {
-  process.env.GranulesTable = `${prefix}-GranulesTable`;
-  const granuleModel = new Granule();
-  return await Promise.all(
+  await Promise.all(
```

Comment:
why `return` is removed?

Reply:
The granule delete endpoint returns information that can be useful for verification purpose
https://github.com/nasa/cumulus/pull/3093/files

Reply:
I remove `returned` because it gives the lint error
```
Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
```